cluster/gce/gci/mounter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
+MAINTAINER vishh@google.com
+
+RUN apt-get update && apt-get install -y nfs-common glusterfs-client
```

Comment:
I'm not sure how we will handle version skew in this scenario. Here is the version in the container this PR introduces. Let's test it and update it and document the supported server versions.

```shell
$ docker run --entrypoint=glusterfs gcr.io/google_containers/gci-mounter:v1 --version
glusterfs 3.5.2 built on May 27 2016 13:18:54
Repository revision: git://git.gluster.com/glusterfs.git
Copyright (c) 2006-2013 Red Hat, Inc. <http://www.redhat.com/>
GlusterFS comes with ABSOLUTELY NO WARRANTY.
It is licensed to you under your choice of the GNU Lesser
General Public License, version 3 or any later version (LGPLv3
or later), or the GNU General Public License, version 2 (GPLv2),
in all cases as published by the Free Software Foundation.
```